src/MSALWrapper.Benchmark/BrokerBenchmark.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.MSALWrapper.Benchmark
+{
+    using BenchmarkDotNet.Attributes;
+    using Microsoft.Authentication.MSALWrapper;
+    using Microsoft.Identity.Client;
+    using Microsoft.Identity.Client.Broker;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Authentication.MSALWrapper.AuthFlow;
+    using BenchmarkDotNet.Running;
+
+    /// <summary>
+    /// <see cref="BrokerBenchmark"/> class is for benchmark comparison between C++ native broker (available after Microsoft.Identity.Client 4.44) and .Net managed broker.
+    /// </summary>
+    public class BrokerBenchmark
+    {
+        private readonly Guid clientID = Guid.Parse("872cd9fa-d31f-45e0-9eab-6e460a02d1f1"); // Visual Studio client id
+        private readonly Guid tenantID = Guid.Parse("72f988bf-86f1-41af-91ab-2d7cd011db47"); // Microsoft "MSIT" corp tenant

```

Comment:
I'm aware that these aren't especially secret values, but could these be CLI arguments rather than hardcoded?

Reply:
Yes. I can add them by introducing CLI utils, but I wonder if it is worth to do because it will make the benchmark more complex.

Reply:
You could leave them as fake Guids (all zeros) to be obvious.

Reply:
@kyle-rader I can't use empty GUID, otherwise I will get error
`00000000-0000-0000-0000-000000000000 is not a valid application identifier.`